packages/test/test-end-to-end-tests/src/test/gc/gcSweepDataStores.spec.ts

```diff
@@ -203,6 +180,7 @@ describeCompat("GC data store sweep tests", "NoCompat", (getTestObjectProvider)
 				{
 					eventName: "fluid:telemetry:FluidDataStoreContext:GC_Deleted_DataStore_Changed",
 					clientType: "noninteractive/summarizer",
+					callSite: "submitMessage",
```

Comment:
I don't see this in the diff above, is this instrumented somewhere else? Or am I just missing it?

Reply:
The callSite already existed. I just added them here because I did the same for the new ones.